### PR TITLE
Remove unnecessary class structures in service

### PR DESCRIPTION
**Pull request by marcelkornblum**
Services are currently implemented as classes across the board. Classes encapsulate state, or allow inheritance or overriding for different contexts. 

Services have none of these requirements and so classes are extraneous. Python modules give the same functional encapsulation as classes, but functions are simpler to invoke and understand and don't make users question whether they may have side effects or other issues. 

This PR moves all service functionality out of classes and into the EX team's standard approach.

> NB Both #39 and #40 depend on this PR

### CODE REVIEW

File: core/services.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from django.contrib.auth import get_user_model
 
-from user.services.user import UserService
+from user import services as user_service
```
Comment (CamLamb) [inline core/services.py:5]: Should the file be called `service` instead of `services`

Comment (CamLamb) [reply core/services.py:5]: Same discussion for other files too

File: user/tests/test_user.py
Comment (CamLamb) [inline user/tests/test_user.py:1]: I feel like the test file name should be `test_service.py` as it exists in the user tests and is main focus is testing the service level functions

File: core/services.py (hunk 1 above)
Comment (marcelkornblum) [reply core/services.py:5]: I also thought similar but... no? 

`views`, `models` are plural and it allows for making a folder with multiple files. If anything the `.. as ..` alias maybe ought to be pluralised

File: user/tests/test_user.py
Comment (marcelkornblum) [reply user/tests/test_user.py:1]: agree but scoping this PR small...

**Pull request merged**